### ISSUE TITLE
TQRespec bug on startup

### ISSUE THREAD

**Issue by neooblaster** [epinter/tqrespec#33]
Hello team,

**Describe the bug**
TQRespec 10.1 can not start even if I put Dabatase & Text File. It asks to set game path.
When I set game path, I get Java error.

***To Reproduce***
My case is specific I suppose but please find below steps : 

1.- Starting up the .EXE. It requests for game location : 

![2021-06-09 13_57_26-Window](https://user-images.githubusercontent.com/22078558/121350528-16e7d180-c92b-11eb-9ca1-f2a5deff43d4.png)

2.- I define path to the game : 

![2021-06-09 13_58_03-](https://user-images.githubusercontent.com/22078558/121350595-2d8e2880-c92b-11eb-9704-be0a71d938e2.png)

3.- It did not succeeded with following Error : 

![2021-06-09 13_58_19-Window](https://user-images.githubusercontent.com/22078558/121350687-3da60800-c92b-11eb-9d84-c2a8767413ce.png)

***Expected behavior***

***Error message (detail)***
````plaintext
br.com.pinter.tqrespec.core.UnhandledRuntimeException: Error loading database.
	at br.com.pinter.tqrespec/br.com.pinter.tqrespec.tqdata.Db.initialize(Db.java:62)
	at br.com.pinter.tqrespec/br.com.pinter.tqrespec.Main$2.call(Main.java:177)
	at br.com.pinter.tqrespec/br.com.pinter.tqrespec.Main$2.call(Main.java:172)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Task.java:1425)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: java.io.FileNotFoundException: Database not found
	at br.com.pinter.tqrespec/br.com.pinter.tqrespec.tqdata.GameInfo.getDatabasePath(GameInfo.java:800)
	at br.com.pinter.tqrespec/br.com.pinter.tqrespec.tqdata.Db.initialize(Db.java:51)
	... 5 more
````

***TQRespec version (please complete the following information):***
- v0.10.1

***Operating System (please complete the following information):***
- Windows 20H2 (install yersterday)

***Game version (please complete the following information):***
- vTQ AE 2.9

***DLCs installed :***
- [X] Ragnarok
- [X] Atlantis

***Additional context***
I already suceeded to start the tool. I don't understand why it ask for my game location and why It crash. Also it must be use local Database file & Text but it also crash. Even if the folder "Gamedata" is empty, the application crash.
I cleared my savegame data folder to have a new fresh folder, but nothing works.
I checked for Microsoft Visual Studio as well. No changes.

Any idea ?

Thanks a lot for your help.

Kindly Regards,
Nicolas.

**Comment by neooblaster** [comment]
Maybe JRE or JDK is required ?
I will try later this afternoon and will keep you informed.

**Comment by epinter** [comment]
Hi,

If the game is not installed, you can run TQRespec copying inside gamedata directory. Follow the instructions from the [README](https://github.com/epinter/tqrespec#running-without-the-game-installed). There must be Text and Database folders inside gamedata, like described in readme. Like:

`
gamedata/Database/database.arz
gamedata/Text/Text_EN.arc
`

**Comment by epinter** [comment]
For some reason, the manual directory chooser (is shown when a installation is not detected) is not working, I will test it.

But you should be able to run TQRespec copying files to gamedata directory.

**Comment by neooblaster** [comment]
> Hi,
> 
> If the game is not installed, you can run TQRespec copying resource directories inside gamedata. Follow the instructions from the [README](https://github.com/epinter/tqrespec#running-without-the-game-installed). There must be Text and Database folders inside gamedata, like described in readme. Like:
> 
> ```
> gamedata/Database/database.arz
> gamedata/Text/Text_EN.arc
> ```

I did not noticed we should keep the folder. I put `Database `& `Text` file in folder `gamedata` instead of respective folders.
Il will try asap. If that works like this, it's OK for me ;)
I will close the thread if that works.

Thank you for your help !

Nicolas.

**Comment by epinter** [comment]
Manual path chooser bug fixed in just released version [v0.10.2](https://github.com/epinter/tqrespec/releases/tag/v0.10.2).

**Comment by neooblaster** [comment]
Hello @epinter ,

I tried to start TQSpec with game data folders `Database` & `Text`. It starts as expected !
I also tried new the version 0.10.2 to check the fixing. I get a large white screen but, I restart it and v0.10.2 did no longer ask for game location and start as expected.

So you fixed my issue and the workaround for those who do not have the game installed works perfectly !

A big thank to you for the support and for the tool !

Kindly Regards,
Nicolas.
